interfaces/builtin/network_setup_control.go

```diff
@@ -45,6 +45,18 @@ const networkSetupControlConnectedPlugAppArmor = `
 
 /run/udev/rules.d/ rw,                 # needed for cloud-init
 /run/udev/rules.d/[0-9]*-netplan-* rw,
+
+capability net_admin,
```

Comment:
Why is net_admin needed if we are talking to netplan over DBus?

Reply:
It's unclear to me why net_admin ends up being necessary, but this is the denial I get: 

```
Aug 14 16:21:57 ian-disco-desktop audit[21051]: AVC apparmor="DENIED" operation="capable" profile="snap.test-snapd-netplan-apply.netplan" pid=21051 comm="busctl" capability=12  capname="net_admin"
Aug 14 16:21:57 ian-disco-desktop kernel: audit: type=1400 audit(1565817717.617:3584): apparmor="DENIED" operation="capable" profile="snap.test-snapd-netplan-apply.netplan" pid=21051 comm="busctl" capability=12  capname="net_admin"
```

`busctl` is being called by netplan here: https://github.com/CanonicalLtd/netplan/blob/5d22e9d22c4a3724d27b80b0cd9b898ae8f59d2b/netplan/cli/commands/apply.py#L55-L60

Note that after more testing, this ends up not being necessary, but busctl tries for it every time.